github/ipfs.yml

```
@@ -747,8 +746,7 @@ repositories:
           /github/ipfs.yml @ipfs/github-mgmt-stewards @ipfs/ipdx
     teams:
       # ATTN: do not add teams with push+ access, use github-mgmt stewards team membership instead
-      maintain:
-        - ipdx # NOTE: ipdx are the creators of GitHub Management framework
+      maintain: []
```

This should be brought back. I'll try to find out why it was removed. If it was because of a comment, we might be missing a `trim()` somewhere.
```suggestion
      maintain:
        - ipdx # NOTE: ipdx are the creators of GitHub Management framework
```